cli/lsp/tsc.rs

```diff
@@ -187,48 +189,115 @@ impl AssetDocument {
   }
 }
 
+fn new_assets_map(
+) -> Arc<Mutex<HashMap<ModuleSpecifier, Option<AssetDocument>>>> {
```

Comment:
That's a mouthful, consider adding `type AssetsMap = HashMap<ModuleSpecifier, Option<AssetDocument>>`